packages/wasm-dpp/test/integration/document/validation/validateDocumentFactory.spec.js

```diff
@@ -87,15 +107,15 @@ describe('validateDocumentFactory', () => {
         rawDocument.$protocolVersion = -1;
 
         const protocolVersionError = new SomeConsensusError('test');
-        const protocolVersionResult = new ValidationResult([
+        const protocolVersionResult = new ValidationResultJs([
           protocolVersionError,
         ]);
 
         validateProtocolVersionMock.returns(protocolVersionResult);
 
-        const result = await validateDocument(rawDocument, dataContract);
+        const result = await validateDocumentJs(rawDocument, dataContractJs);
```

Comment:
What is the point of calling old JS validation here?

Reply:
good catch it shouldn't be here